_data/assetPaths.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Should I have added this? Should we gitignore it?

Reply:
We shouldn't add it but we also can't put it into `.gitignore` because it causes style build issues. What I've done is put it into my git `excludes` list. Here is a write up I made for get.gov: 

We want to avoid commiting the `assetPaths.json` file, but need to keep it out of the project `.gitignore` in order to allow eleventy to rebuild when it is changed. One solution to resolve this you issue is to add `assetPaths.json` to the git exclude list:
1. Open up `.git/info/exclude`
2. Add `assetPaths.json` to that file


Thanks for catching this, I should have put this in the replatforming documentation. I'll add it.